qa/smoke-test-http/src/javaRestTest/java/org/elasticsearch/http/BlockedSearcherRestCancellationTestCase.java

```diff
@@ -71,11 +72,16 @@ protected boolean addMockInternalEngine() {
         return false;
     }
 
-    void runTest(Request request, String actionPrefix) throws Exception {
+    void runTest(String testName, Request request, String actionPrefix) throws Exception {
```

Comment:
Rather than passing in a string that (hopefully) identifies the test, I'd suggest using `org.apache.lucene.tests.util.LuceneTestCase#getTestName` (and maybe `org.apache.lucene.tests.util.LuceneTestCase#getTestClass`).